protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -640,4 +641,5 @@ struct AppStorage {
     mapping(IERC20 => uint256) internalTokenBalanceTotal;
     uint256 fertilizedPaidIndex;
     uint256 plenty;
```

Comment:
With the secure beanstalk update, beanstalk now stores the total plenty per asset, so that invariable does not fail. We will need to update this plenty variable to be on a per well basis, and also update this variable upon an plenty interaction (i.e, during a flood, and when a user claims.)

Reply:
Good call, implemented here, lmk if this looks correct: https://github.com/BeanstalkFarms/Beanstalk/commit/ae19fe2cb913a9473ee34ab22f54533aa6a237e6